frontend-react/src/pages/TermsOfService.tsx

```diff
@@ -3,475 +3,520 @@ import { Alert } from "@trussworks/react-uswds";
 export const TermsOfService = () => {
     return (
         <>
-            <section className="usa-site-alert usa-site-alert--info margin-top-neg-4 desktop:margin-top-neg-8 margin-bottom-3 tablet:margin-bottom-6" id="anchor-top">
+            <section
```

Comment:
This file seems like it was a prettier change only?
This is going to make merging with other branches _really_ painful.